tests/cache/tests.py

```diff
@@ -1050,6 +1050,13 @@ def test_createcachetable_with_table_argument(self):
         )
         self.assertEqual(out.getvalue(), "Cache table 'test cache table' created.\n")
 
+    def test_num_queries_of_delete_many(self):
+        cache.set('a', 'a')
+        cache.set('b', 'b')
+        cache.set('c', 'c')
+        with self.assertNumQueries(1):
+            cache.delete_many(['a', 'b', 'c'])
+
```

Comment:
could also assert they are all gone

Also *if* we leave 1 item with `=` as a special case (which I don't believe is necessary, as I wrote above), it will need its own test for 100% coverage

Reply:
There is a test to check if they are all gone in `class BaseCacheTests`.
So it is not needed in this

Reply:
you're right!